<launch>
    <!-- TODO: Maybe change to something better by default? -->
    <arg name="dir" default="~" />
    <arg name="factor" default="-1" />
    <arg name="hfactor" default="$(arg factor)" />
    <arg name="wfactor" default="$(arg factor)" />
    <arg name="bgpercentile" default="0.5" />
    <arg name="subbg" default="false" />
    <!--<arg name="track2d" default="/mdl_people_tracker/tracked_persons_2d" />-->
    <!--<arg name="rgb" default="/head_xtion/rgb/image_rect_color" />-->
    <!--<arg name="d" default="" />-->
    <arg name="src" default="rear" />
    <arg name="track2d" default="/spencer/perception_internal/people_tracking/rgbd_$(arg src)_top/tracked_persons_2d" />
    <arg name="rgb" default="/spencer/sensors/rgbd_$(arg src)_top/rgb/image_raw" />
    <arg name="d" default="/spencer/sensors/rgbd_$(arg src)_top/depth/image_rect" />

    <node pkg="biternion" type="dump_tracks" name="dump_tracks" output="screen">
        <param name="dir" value="$(arg dir)" />
        <param name="hfactor" value="$(arg hfactor)" />
        <param name="wfactor" value="$(arg wfactor)" />
        <param name="bgpercentile" value="$(arg bgpercentile)" />
        <param name="subbg" value="$(arg subbg)" />
        <param name="track2d" value="$(arg track2d)" />
        <param name="rgb" value="$(arg rgb)" />
        <param name="d" value="$(arg d)" />
    </node>
</launch>
